recipes/sfml/all/patches/0002-allow-non-x86-64-macos.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 98b3ffd7..3a082061 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -253,7 +253,7 @@ if(SFML_OS_MACOSX)
+     endif()
+ 
+     # only the default architecture (i.e. 64-bit) is supported
+-    if(NOT CMAKE_OSX_ARCHITECTURES STREQUAL "x86_64")
++    if(0)
+         message(FATAL_ERROR "Only 64-bit architecture is supported")
+     endif()
+ 
```

Comment:
I don't think it's wise to enable a platform that is not officially supported. Does the library work on non x86-64 macos ?

Reply:
It compiles and link, maybe someone with macOS M1 can confirm that SFML properly works.